efficientdet/keras/segmentation.py

```diff
@@ -0,0 +1,136 @@
+from keras.efficientdet_keras import EfficientDetNet
+from utils import activation_fn
+import tensorflow as tf
+
+NUM_CLASSES = 3
+
+
+class EfficientDetSegmentation(EfficientDetNet):
+
+  def build(self, input_shape):
+    self.con2d_ts = []
+    self.bns = []
+    for _ in range(self.config.max_level - self.config.min_level):
+      self.con2d_ts.append(
+          tf.keras.layers.Conv2DTranspose(
```

Comment:
Should we create them in __init__ instead? (since they are input shape independent).